scripts/index.js

```diff
@@ -0,0 +1,74 @@
+const setIndicatorColor = (tag, value) => {
+    if (value >= 80) {
+        tag.style.color = "red";
```

Comment:
Consider refactoring to assign a className, and then put the styles in the `css` file. For example:

```
#index.js
tag.className = “red”;

#styles.css
.red {
	color: red;
}
```